dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -29,7 +29,8 @@ def current
         under_13: current_user.under_13?,
         over_21: current_user.over_21?,
         sort_by_family_name: current_user.sort_by_family_name?,
-        show_progress_table_v2: current_user.show_progress_table_v2
+        show_progress_table_v2: current_user.show_progress_table_v2,
+        progress_table_v2_closed_beta: current_user.progress_table_v2_closed_beta?
```

Comment:
One of these has a `?` and one doesn't. Is one of them incorrect or are both valid? Should probably follow a consistent style regardless.